cicecore/cicedynB/dynamics/ice_dyn_evp.F90

```diff
@@ -1824,73 +1756,77 @@ subroutine stressC_U  (nx_block, ny_block,  &
       integer (kind=int_kind) :: &
          i, j, ij
 
+      real (kind=dbl_kind), dimension (nx_block,ny_block) :: &
+         DeltaU       ! strain rates at U point
+
       real (kind=dbl_kind) :: &
-        divU, tensionU, shearU, DeltaU, & ! strain rates at U point
-        zetax2U, etax2U, rep_prsU,      & ! replacement pressure at U point
-        DminUarea, strtmp, areatmp        ! Dmin on U and tmp variables
+         zetax2U, etax2U, rep_prsU,      & ! replacement pressure at U point
+         DminUarea, strtmp, areatmp        ! Dmin on U and tmp variables
 
       character(len=*), parameter :: subname = '(stressC_U)'
       
+      !-----------------------------------------------------------------
+      ! strain rates at U point
+      ! NOTE these are actually strain rates * area  (m^2/s)
+      !-----------------------------------------------------------------
+
+      if (visc_coeff_method == 'avg_strength') then
```

Comment:
this could be done later but we could change visc_coeff_method by visc_method or viscosity_method

Reply:
Good point, we'll fix this next.